.dockerignore

```diff
@@ -3,6 +3,8 @@
 *.iml
 *.md
 *.yaml
+.devcontainer
+.git
```

Comment:
Excluding `git`  is a bigger change than it looks. The `.git` folder gets modified every time your run `git` preventing caching of builds from working. Once removed, we must pass `GIT_COMMIT` as Docker build-args.